finesse/hardware/temperature/dummy_dp9800.py

```diff
@@ -0,0 +1,50 @@
+"""This module provides an interface to dummy DP9800 temperature readers."""
+import logging
+
+from pubsub import pub
+
+from .dp9800 import DP9800, DP9800Error
+
+
+class DummyDP9800(DP9800):
```

Comment:
As I said before, I'd rewrite this class so that it doesn't use the `DP9800` parser at all. These various bits of state make it unnecessarily complicated and bugprone.

Just have a member function called `read_temperatures` for both `DP9800` and `DummyDP9800` and the dummy version can just return static data (or random data).